#! /bin/sh -e
 
# DP: Description: Fix ustat.c compile error for s390
# DP: Author: Gerhard Tonn <GerhardTonn@gammatau.de>
# DP: Upstream status: Not submitted
# DP: Status Details: 
# DP: Date: 23 Mar 2003
 
if [ $# -ne 2 ]; then
    echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
    exit 1
fi
case "$1" in
    -patch) patch -d "$2" -f --no-backup-if-mismatch -p0 < $0;;
    -unpatch) patch -d "$2" -f --no-backup-if-mismatch -R -p0 < $0;;
    *)
        echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
        exit 1
esac
exit 0

--- linuxthreads/sysdeps/s390/tls.h.bak	2003-03-23 08:59:35.000000000 +0000
+++ linuxthreads/sysdeps/s390/tls.h	2003-03-23 09:47:49.000000000 +0000
@@ -124,7 +124,7 @@
 # ifndef __ASSEMBLER__
 
 /* Get the thread descriptor definition.  */
-#  include <linuxthreads/descr.h>
+//#  include <linuxthreads/descr.h>
 
 #  define NONTLS_INIT_TP \
   do { 								\
